Tasks/KubernetesManifestV0/task.json

```diff
@@ -79,6 +92,19 @@
                 "message": "Enter valid percentage value i.e between 0 to 100."
             }
         },
+        {
+            "name": "baselineAndCanaryReplicas",
+            "type": "string",
+            "label": "Baseline And Canary Replicas",
```

Comment:
Do we need to take 'percentage' input for TrafficSplit method? As I can see in this case, we are not using percentage.

Reply:
we are using while creating the traffic split object.